Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -3029,7 +3038,7 @@ extension BrowserViewController: ContextMenuHelperDelegate {
                     }
                     accessDenied.addAction(settingsAction)
                     self.present(accessDenied, animated: true, completion: nil)
-
+                    LeanplumIntegration.sharedInstance.track(eventName: .downloadedImage)
```

Comment:
Everywhere in the app we call this *Save Image*. Why is the event name *Download Image* ?